tests/baselines/reference/documentHighlightAtInheritedProperties1.baseline.jsonc

```diff
@@ -0,0 +1,14 @@
+// === /tests/cases/fourslash/file1.ts ===
+// interface interface1 extends interface1 {
+//    /*HIGHLIGHTS*/<|[|{| contextId: 0, kind: "reference" |}doStuff|](): void;|>
+//    propName: string;
+// }
+
+
+
+// === /tests/cases/fourslash/file1.ts ===
+// interface interface1 extends interface1 {
+//    doStuff(): void;
+//    /*HIGHLIGHTS*/<|[|{| contextId: 0, kind: "reference" |}propName|]: string;|>
```

Comment:
Can we at least drop the `contextId` when it’s not needed to disambiguate anything?

Also, I think each baseline output in this file could use a “document highlights” heading to make it more clear that these are two separate invocations of the command. I like how some of the other baselines are divided up with `goToDefinition`, `goToType`, etc. headings. I think it would be good if that was consistent across all baselines—any time a baseline-producing command is run, the title gets emitted.